cypress/support/pages/profile.pageObject.js

```diff
@@ -0,0 +1,16 @@
+import PageObject from '../PageObject';
+
+class profilePageObject extends PageObject {
```

Comment:
Class name should be in PascalCase. Rename 'profilePageObject' to 'ProfilePageObject'

Reply:
done